pkg/policy/policy.go

```diff
@@ -27,12 +27,13 @@ const DefaultVersion = "2012-10-17"
 
 // Args - arguments to policy to check whether it is allowed
 type Args struct {
-	AccountName     string
-	Action          Action
-	BucketName      string
-	ConditionValues map[string][]string
-	IsOwner         bool
-	ObjectName      string
+	AccountName     string                 `json:"account"`
+	Action          Action                 `json:"action"`
+	BucketName      string                 `json:"bucket"`
+	ConditionValues map[string][]string    `json:"conditions"`
+	IsOwner         bool                   `json:"owner"`
+	ObjectName      string                 `json:"object"`
+	Claims          map[string]interface{} `json:"claims"`
```

Comment:
this field can be declared as `jwtgo.MapClaims`

Reply:
That incurs dependency @krishnasrinivas I want to explicitly avoid that since its a package...

Reply:
I wanted to keep it basic type, not specific to jwt package.